katello/katello-backup

```diff
@@ -111,6 +111,8 @@ else
     '/etc/pki/pulp',
     '/etc/pulp',
     '/etc/puppet',
+    '/etc/puppetlabs/puppet/ssl',
+    '/etc/puppetlabs/puppet/foreman_cache_data',
```

Comment:
I agreed with @stbenjam, we need to grab all of the /etc/puppetlabs directory. this directory contains puppet codes as well.